plugins/parodos/src/models/project.ts

```diff
@@ -8,17 +8,26 @@ const projectStatus = z.union([
   z.literal('COMPLETED'),
 ]);
 
+const accessRole = z.union([
+  z.literal('Owner'),
+  z.literal('Developer'),
+  z.literal('Admin'),
+]);
+
 export type ProjectStatus = z.infer<typeof projectStatus>;
 
 export const projectSchema = z.object({
   id: z.string(),
   name: z.string(),
   description: z.string().nullable(),
-  createDate: z.coerce.date(),
-  modifyDate: z.coerce.date(),
+  createdDate: z.coerce.date(),
+  modifiedDate: z.coerce.date(),
   status: projectStatus
     .default('IN_PROGRESS')
     .transform(value => value.split('_').join(' ')),
+  modifiedBy: z.string().nullable(),
```

Comment:
```suggestion
  modifiedBy: z.string().nullable().optional(),
```

Everything is required by default unless  `optional` is specified.

Reply:
ack